webclient/app/src/app/features/apps/entities/entityDialog/EntityDialog.jsx

```diff
@@ -7,6 +7,7 @@ import {
     DialogActions,
```

Comment:
Was war der Anlass / Fehler der ganzen Änderungen hier?

Reply:
Durch die JHipster Sache ist es nicht ganz so schick gewesen die Liste der Elemente in dem Entity zu kontrollieren. Deshalb musste ich erstmal parsen, ob das required in der Liste ist, damit ich das in ein passendes Boolean Wert übersetzen kann (ist dadurch deutlich leichter mit zu arbeiten). 
Die restlichen Sachen war dann duplizierter code, den ich durch die ausgelagerte Funktion eingespart habe.